lattice_planner_node:
  ros__parameters:
    planner_former_point_of_current_position: 0 # 这个值可以将启始点向前推进一定的点的数量，但是不能太多，否则会出现找点错误的问题。

    planner_working_mode: 1 # 1: simulation 2: real vehicle.

    c_speed : 0.0
    c_d : 0.0
    c_d_d : 0.0
    c_d_dd : 0.0
    s0 : 0.0